firebase/rules.bolt

```diff
@@ -85,6 +92,10 @@ getUserId() {
   ('user-' + auth.uid)
 }
 
+getPrivateLifeId() {
+  ('privateLife-' + auth.uid)  
```

Comment:
alas, we cant make this assumption =(

a player might go through multiple lives throughout the game, and each one will have a different ID, and a different code.